lib/ansible/cli/galaxy.py

```diff
@@ -239,10 +241,21 @@ def add_init_options(self, parser, parents=None):
                                           "'apb' and 'network'.")
 
     def add_remove_options(self, parser, parents=None):
-        remove_parser = parser.add_parser('remove', parents=parents, help='Delete roles from roles_path.')
-        remove_parser.set_defaults(func=self.execute_remove)
+        if parser.metavar == 'COLLECTION_ACTION':
+            remove_parser = parser.add_parser('remove', parents=parents, help='Delete collections installed in the collections path')
```

Comment:
@s-hertel did we talk about `remove` vs `uninstall`? It occurred to me that people switching from pip may expect `uninstall`.

Reply:
To be honest this was the first thing I tried before finding out that this wasn't something in ansible-galaxy yet